examples/with-auth/README.md

```diff
@@ -0,0 +1,82 @@
+## Feature Example - Authentication
+
+This project demonstrates how to add password based authentication to your Keystone system.
+It builds on the [Task Manager](../todo) starter project.
+
+## Instructions
+
+To run this project, clone the Keystone repository locally then navigate to this directory and run:
+
+```shell
+yarn dev
+```
+
+This will start the Admin UI at [localhost:3000](http://localhost:3000).
+You can use the Admin UI to create items in your database.
+
+You can also access a GraphQL Playground at [localhost:3000/api/graphql](http://localhost:3000/api/graphql), which allows you to directly run GraphQL queries and mutations.
+
+## Features
+
+This project demonstrates how to add authentication to a Keystone system.
+We're going to use the [`@keystone-next/auth`](https://next.keystonejs.com/apis/auth) package, along with Keystone's [session management API](https://next.keystonejs.com/apis/session), to add the following features to the base system:
+
+ * Configure which fields to use for signin
+ * Set up stateless session handling to keep track of the signed in user
+ * Add a signin screen to the Admin UI
+ * Add a signout button to the Admin UI
+ * Allow the signed in user to access their own details in the Admin UI
+ * Add a helper page to the Admin UI to allow you to create your first user when starting from an empty database
+
+### Added fields
+
+In this example we add two new fields, `email` and `password`, to the `Person` list.
+These are used as our _identity_ and _secret_ fields for login.
+
+```typescript
+    email: text({ isRequired: true, isUnique: true }),
+    password: password({ isRequired: true }),
+```
+
+### Auth config
+
+We use the `createAuth` function from `@keystone-next/auth` to configure a `withAuth` config wrapper, which will inject all the extra config used to enabled configuration.
+
+```typescript
+import { createAuth } from '@keystone-next/auth';
+
+const { withAuth } = createAuth({
+  listKey: 'Person',
+  identityField: 'email',
+  secretField: 'password',
+  initFirstItem: { fields: ['name', 'email', 'password'] },
+});
+```
+
+### Session
+
+We use a basic `statelessSessions` from `@keystone-next/keystone/session` for session handling.
```

Comment:
@timleslie  are there existing docs we can link to here that explains what `statelessSession` does?